index.html

```diff
@@ -5,7 +5,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="styles/base.css" type="text/css" rel="stylesheet">
     <link href="styles/layout.css" type="text/css" rel="stylesheet">
+    <link href="styles/modules.css" type="text/css" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+    <script id="template" type="text/x-handlebars-template">
+    <article>
+      <p id="name">Project: <a target="_blank" href="{{path}}">{{name}}</a></p>
+      <p id="description">{{description}}</p>
+      <p id="languages">Langages Used: {{languages}}</p>
```

Comment:
Nice work using Handlebars template expressions to represent your model properties in HTML :)